app/controllers/pages_controller.rb

```diff
@@ -0,0 +1,5 @@
+class PagesController < ApplicationController
+  def admin
+    @admin_members = SlackUser.admins
```

Comment:
I think it might also be nice to sort the admins by "most recent chat first" under the theory that those admins that have chatted recently are more likely to be around.

Can we also add `where.not(last_message_at: nil).order(last_message_at: :desc)` into that query?